InstantGrinder/InstantGrinder/InstantGrinderPlugin.cs

```diff
@@ -72,27 +99,89 @@ public bool TryGetPlayerByName(string playerName, out MyPlayer player)
             return false;
         }
 
+        private void PlayerRefresh(MyPlayer player)
+        {
+            var playerEndpoint = new Endpoint(player.Id.SteamId, 0);
+            var replicationServer = (MyReplicationServer)MyMultiplayer.ReplicationLayer;
+            object clientData;
+
+            try
+            {
+                clientData = ClientStates.Invoke(replicationServer)[playerEndpoint];
+            }
+            catch
+            {
+                return;
+            }
```

Comment:
Should explicitly fail or output an error message instead of discarding an error. It'll be difficult to track down the issue down the road